<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RetroMan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="scanButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan library</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scanDeviceButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scrapeButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Scrape</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Library path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Device name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="friendlyNameTextField">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>491</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="romsPathTextField">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
